### ISSUE TITLE
i18n: NEXT_LOCALE cookie being ignored

### ISSUE THREAD

**Issue by Lunnatica** [serverless-nextjs/serverless-next.js#1106]
### Describe the bug

When setting up a NEXT_LOCALE cookie, the cookie is being ignored and the locale language is not overridden.

This works in localhost, but nothing happens when it's deployed to production.

Cloudfront is receiving the accept_locale header and I've also whitelisted the NEXT_LOCALE cookie.

### Actual behavior
- Set up browser language to Spanish.
- A NEXT_LOCALE cookie is set up in the client browser with value 'en'
- Go to root url  (/)
- The page redirects to the locale version (/es)

### Expected behavior
- A cookie NEXT_LOCALE cookie is set up in the client browser with value 'en'
- Go to root url 
- The page stays in the root (/)

### Steps to reproduce

- Set up browser language to Spanish.
- Go to page with configured locales for 'en' and 'es' (default en).

```
 i18n: {
            locales: ['en', 'es'],
            defaultLocale: 'en',
        },
```

- Page redirects to /es
- Set up NEXT_LOCALE cookie to en (e.g. `document.cookie = 'NEXT_LOCALE=en'` in the browser console).
- Go to root URL (/).
- Page is still being redirected to /es

### Versions

- @sls-next/serverless-component version: @sls-next/serverless-component@1.19.0
- Next.js version: 10.2.2 (but I've tried downgrading to v10.0.0 too and it's still the same behaviour).

### Additional context

Cloudfront cookie settings:

```
cloudfront:
    distributionId: <ID>
    defaults:
        forward:
            cookies: 'NEXT_LOCALE'
```

**Comment by dphang** [comment]
Yes I think it was missed (not sure how I missed in the docs). We will need to check NEXT_LOCALE cookie as well, I will look to fix for the 1.20 release

**Comment by dphang** [comment]
Should be fixed by linked PR.
